.circleci/config.yml

```diff
@@ -203,6 +209,9 @@ jobs:
             mkdir cobertura
             bundle exec slather
           working_directory: ./auth0_flutter/example/ios
+      - run:
+          name: Create ~/.gnupg dir if it does not exist
+          command: mkdir -p ~/.gnupg
```

Comment:
Apparently the image used for M1 builds does not have a `.gnupg` dir. See https://github.com/codecov/codecov-circleci-orb/issues/157